curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f5486b8fd4b71633f69b0.md

```diff
@@ -29,16 +29,16 @@ Your `p` element should come after your `.small-text` element.
 assert(document.querySelector('.small-text')?.nextElementSibling?.localName === 'p');
 ```
 
-Your `p` element should have the text `Calories`.
+Your `span` element should come after your `.left-container` element.
 
 ```js
-assert(document.querySelector('.left-container')?.lastElementChild?.innerText === 'Calories');
+assert(document.querySelector('.left-container')?.nextElementSibling?.localName ==='span');
 ```
 
-Your `span` element should come after your `.left-container` element.
+Your `p` element should have the text `Calories`.
 
 ```js
-assert(document.querySelector('.left-container')?.nextElementSibling?.localName ==='span');
+assert(document.querySelector('.left-container')?.lastElementChild?.innerText === 'Calories');
```

Comment:
We can also 'belt-and-suspenders' this so the check for the contents for p match the test for the location of p, but I still think checking the location of both *first* is less fragile
```suggestion
assert(document.querySelector('.left-container')?.nextElementSibling?.innerText === 'Calories');
```